### ISSUE TITLE
could you release the training code?

### ISSUE THREAD

**Issue by chenping5121** [wangruixing/DeepUPE#50]
I construct loss function—color loss by myself, but I don't know whether it's correct or not. And it's confused me that the total loss is reduced first and increased when training. So can you release the training code so that I can know where is the problem in my training code?

**Comment by lycutter** [comment]
> I construct loss function—color loss by myself, but I don't know whether it's correct or not. And it's confused me that the total loss is reduced first and increased when training. So can you release the training code so that I can know where is the problem in my training code?

Hello, I wonder how you construct the color loss in your code. Thank you.

**Comment by lycutter** [comment]
> I construct loss function—color loss by myself, but I don't know whether it's correct or not. And it's confused me that the total loss is reduced first and increased when training. So can you release the training code so that I can know where is the problem in my training code?

Hello, I wonder if you could release your code. I wonder how you constrcut the color loss. Thanks!

**Comment by wangruixing** [comment]
> > I construct loss function—color loss by myself, but I don't know whether it's correct or not. And it's confused me that the total loss is reduced first and increased when training. So can you release the training code so that I can know where is the problem in my training code?
> 
> Hello, I wonder if you could release your code. I wonder how you constrcut the color loss. Thanks!

Please contact me by email, rxwang@cse.cuhk.edu.hk, I will send you the code and help you to reproduce our result.
